### ISSUE TITLE
Where to find more articles to read on Neural Net debugging scenarios?

### ISSUE THREAD

**Issue by thak123** [allenai/writing-code-for-nlp-research-emnlp2018#1]
Hi,

Thanks for the wonderful write up. There is one section in the PDF which deals with high gradient values and using gradient accumulation for solving the same. 

Where can I learn more about such scenarios and their handling ?
